### DISCUSSION TITLE
Videos creates lots of bonus thumbnails

### DISCUSSION THREAD

**Discussion by T-prog3** [TumblThreeApp/TumblThree#314] [Q&A (Question & Answer)]
When downloading from twitter blogs you get lots of bonus thumbnail images of videos you download. For example, every animated meme video have both a thumbnail image and a video downloaded with the same name. these videos that normally would be .GIF images.

Longer videos both with and without sound also have thumbnails but they have a different name from the video and are blurry and pixelated.

-----

![ERuH-c_VUAE2GW0](https://user-images.githubusercontent.com/74016497/155303705-ae684e2e-5bcd-416f-b664-f43cef8784c1.jpg)

https://user-images.githubusercontent.com/74016497/155303707-c012cc45-4ac0-4dcf-ad2b-dbfd3ce76305.mp4

![ERVMHXaUYAASr1q](https://user-images.githubusercontent.com/74016497/155303841-12f8439f-b0ab-4500-8692-b64507194f28.jpg)

https://user-images.githubusercontent.com/74016497/155303846-b2e2eec3-b627-4706-a078-de3f655c7d63.mp4

**Comment by thomas694** [comment]
Twitter converts GIF files to videos to save bandwidth among others.

The corn still looks crisp. Doesn't the blurriness and pixelation depend on the video it was taken from?
Anyways, the filename should be the same like the video or at least start with the video filename, so that they are not only grouped by date but same start of the filename too.

So what would be a good solution? No thumbnails for GIF 'images' and optional ones for video files? None at all?

**Comment by T-prog3** [comment]
Both pictures above is from GIF to video files. It's true that blurriness and pixelation depend on the video but more precise the first frame of the video and usually the first frames is out of focus.

Filenames is grouped on GIF-Video files but not on regular video files. 

Gif:
![E3Y2uTZXIAQbPSQ](https://user-images.githubusercontent.com/74016497/156743887-78ca00ae-bd4d-4d2b-b344-a42f8b021e95.jpg)

https://user-images.githubusercontent.com/74016497/156743888-04fc32e8-6f6c-4b95-a402-8412afd678ee.mp4

The filenames of GIF-videos: `E3Y2uTZXIAQbPSQ.jpg` and `E3Y2uTZXIAQbPSQ.mp4`

-----------------------------------------------------------------------
Video:
https://user-images.githubusercontent.com/74016497/156744572-55c1b326-01c2-4d2c-a42f-848a905bcf72.mp4

![Vz-SK9zlz1HHUjiA](https://user-images.githubusercontent.com/74016497/156744221-cf3d5e83-dfbc-4100-bd19-cb21dc9e002d.jpg)

The filenames of videos: `OjABGpYZev_oPdRd.mp4` and `6iBkelCLy3BefLAz.jpg`


I would vote for none at all but i guess that the best solution would be to have thumbnails as an optional setting and not default. But i can't imagine anyone that would want thumbnails of GIF-Video files. As of now i have about 20k of GIF-video thumbnails that needs to be removed.

**Comment by thomas694** [comment]
No more thumbnails for GIFs and new option for video thumbnails.

**Comment by thomas694** [comment]
The issue has been fixed and closed. You can still comment. Feel free to ask for reopening the issue if needed.
